opensrp-giz-malawi/src/main/java/org/smartregister/giz/fragment/FilterReportFragment.java

```diff
@@ -108,6 +124,7 @@ private void bindAutoCompleteText() {
         autoCompleteTextView.setOnTouchListener(new View.OnTouchListener() {
             public boolean onTouch(View v, MotionEvent event) {
                 ((InputMethodManager) (context).getSystemService(Context.INPUT_METHOD_SERVICE)).toggleSoftInput(InputMethodManager.SHOW_FORCED, InputMethodManager.HIDE_IMPLICIT_ONLY);
+                autoCompleteTextView.requestFocus();
```

Comment:
I think `v.requestFocus()` should work fine,

Reply:
Replaced